omniscidb/OSDependent/Windows/CMakeLists.txt

```diff
@@ -1 +1,4 @@
 add_library(OSDependent ${OSDEPENDENT_SOURCE_FILES})
+set_target_properties(OSDependent PROPERTIES LIBRARY_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/..)
+
+target_link_libraries(OSDependent Logger)
```

Comment:
IMO, Logger should be a private dependency here as well as in the device managers. What made you decide otherwise?

Reply:
I haven't made any decisions and simply copied these lines from `Unix/CMakeLists.txt`.

Reply:
I guess it doesn't hurt too much, we'll deal with it separately.